apps/ccf-rui/src/app/modules/content/content.component.ts

```diff
@@ -80,36 +82,24 @@ export class ContentComponent implements OnInit, OnDestroy {
    */
   showDebugButtons = !environment.production;
 
-  /** Resize detection */
-  private sensor!: ResizeSensor;
-
   /**
-   * Sets up the resize sensor
+   * Updates the gizmo rotation based on the scene rotation when not in 3D view
    */
   ngOnInit(): void {
-    this.sensor = new ResizeSensor(this.rootRef.nativeElement, ({ width }) => {
-      const isNarrowView = width < 440; // 27.5rem
-      if (this.isNarrowView !== isNarrowView) {
-        this.isNarrowView = isNarrowView;
-        this.cdr.markForCheck();
+    combineLatest([this.is3DView$, this.scene.rotation$]).subscribe(([is3D, rotation]) => {
+      if (!is3D) {
+        this.setGizmoRotation([rotation, 0]);
       }
     });
   }
 
   /**
-   * Detaches the resize sensor
-   */
-  ngOnDestroy(): void {
-    this.sensor.detach();
-  }
-
-  /**
-   * Sets view type
-   *
-   * @param is3DView Set view type to '3d' if this is true otherwise set it to 'register'
+   * Sets gizmo rotation
+   * @param rotation Rotation values
    */
-  setViewType(is3DView: boolean): void {
-    this.model.setViewType(is3DView ? '3d' : 'register');
+  setGizmoRotation(rotation: [number, number]): void {
+    this.gizmo().rotation = rotation[0];
```

Comment:
Store the rotation in signals then bind it in the template as inputs instead of setting it directly